### ISSUE TITLE
Ansible dependency error post tendrl-commons install

### ISSUE THREAD

**Issue by andrewlove** [Tendrl/commons#444]
After installing tendrl-commons any further yum update gives:

```root@ose-gluster-001 yum.repos.d]# yum update
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-
              : manager
This system is not registered with Subscription Management. You can use subscription-manager to register.
Loading mirror speeds from cached hostfile
 * base: mirror.econdc.com
 * epel: mirror.freethought-internet.co.uk
 * extras: mirror.freethought-internet.co.uk
 * updates: mirror.freethought-internet.co.uk
Resolving Dependencies
--> Running transaction check
---> Package ansible.noarch 0:2.2.1.0-2.el7.centos will be updated
--> Processing Dependency: ansible = 2.2.1.0 for package: tendrl-commons-1.2.3-05_07_2017_05_13_23.noarch
---> Package ansible.noarch 0:2.3.0.0-3.el7 will be an update
--> Finished Dependency Resolution
Error: Package: tendrl-commons-1.2.3-05_07_2017_05_13_23.noarch (@tendrl-tendrl)
[root@ose-gluster-001 yum.repos.d]# yum update
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-
              : manager
This system is not registered with Subscription Management. You can use subscription-manager to register.
Loading mirror speeds from cached hostfile
 * base: mirror.econdc.com
 * epel: mirror.freethought-internet.co.uk
 * extras: mirror.freethought-internet.co.uk
 * updates: mirror.freethought-internet.co.uk
Resolving Dependencies                                                               --> Running transaction check
---> Package ansible.noarch 0:2.2.1.0-2.el7.centos will be updated
--> Processing Dependency: ansible = 2.2.1.0 for package: tendrl-commons-1.2.3-05_07_2017_05_13_23.noarch
---> Package ansible.noarch 0:2.3.0.0-3.el7 will be an update
--> Finished Dependency Resolution
Error: Package: tendrl-commons-1.2.3-05_07_2017_05_13_23.noarch (@tendrl-tendrl)
           Requires: ansible = 2.2.1.0
           Removing: ansible-2.2.1.0-2.el7.centos.noarch (@tendrl-dependencies)
               ansible = 2.2.1.0-2.el7.centos                                                   Updated By: ansible-2.3.0.0-3.el7.noarch (epel)
               ansible = 2.3.0.0-3.el7
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@ose-gluster-001 yum.repos.d]#```

**Comment by andrewlove** [comment]
Duplicate of #436 
Closing
